.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: csharp
+
+matrix:
+  include:
+    - os: linux # Ubuntu 14.04
+      dist: trusty
+      sudo: required
+      dotnet: 1.0.0-preview2-003121
```

Comment:
Travis supports only this two versions:
1.0.0-preview2-003121	Microsoft .NET Core 1.0.0 - SDK Preview 2
1.0.0-preview2-003131	Microsoft .NET Core 1.0.1 - SDK 1.0.0 Preview 2

I can update to .NET Core 1.0.1